lib/angular-smooth-scroll.js

```diff
@@ -114,7 +118,12 @@
 			// Stop the scrolling animation when the anchor is reached (or at the top/bottom of the page)
 			var stopAnimation = function () {
 				var currentLocation = getScrollLocation();
-				if ( position == endLocation || currentLocation == endLocation || ( (window.innerHeight + currentLocation) >= document.body.scrollHeight ) ) {
+				var containerHeight = container.offsetHeight;
```

Comment:
I changed the code here to
```javascript
var containerHeight = container ? container.offsetHeight : window.innerHeight;
var containerScrollHeight = container ?  container.scrollHeight : document.body.scrollHeight;

// Depending on the scroll direction check if we reached the top or bottom
var endCondition = distance < 0 ? currentLocation === 0 :
            (containerHeight + currentLocation) >= containerScrollHeight;
```
That fixes the access to container even though it might be null and scrolling upwards.